libs/game/sprite.ts

```diff
@@ -130,6 +132,27 @@ class Sprite extends sprites.BaseSprite {
         this._ay = Fx8(v)
     }
 
+    //% group="Physics" blockSetVariable="mySprite"
+    //% blockCombine block="fx (friction x)" callInDebugger
+    get fx(): number {
+        return Fx.toFloat(this._fx)
+    }
+    //% group="Physics" blockSetVariable="mySprite"
+    //% blockCombine block="fx (friction x)"
+    set fx(v: number) {
+        this._fx = Fx8(Math.max(0, v))
```

Comment:
Hm, negative friction for "slipperyness" is almost interesting, much more reasonable to just not allow it though

Reply:
It will lead to confusing, and fun, behavior but mostly confusing bugs.